wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/Views/Components/DeviceDetailsProteusView.swift

```diff
@@ -22,6 +22,7 @@ import WireCommonComponents
 struct DeviceDetailsProteusView: View {
     @ObservedObject var viewModel: DeviceInfoViewModel
     @State var isVerfied: Bool
+    var shouldShowActivatedDate: Bool = true
```

Comment:
Can this be private scoped?

Reply:
Nope. This needs to be at the same level to update this value from `ProfileDeviceDetailsView`